deepfence_ui/app/scripts/components/vulnerability-view/vulnerability-table-view/vulnerability-table-v2-view.js

```diff
@@ -212,10 +215,18 @@ class VulnerabilityTableV2 extends React.Component {
       scan_id: this.props.scanId,
     };
 
+    let from = page ? page * pageSize : page
+
+    if(this.state.deletedValues.length > 0) {
```

Comment:
Does it cover the followings:
Lets say 4 pages.
i) delete few or all from first page
ii) delete few or all from second, third page
iii) delete few or all from last page
iv) select few or all but no delete and  just the navigation to back and forth of pages and check the scenarios.

Reply:
> Could you please add comment on what is doing by this code?

We are doing backend pagination normally. If there are entries deleted, we check for the number of entries deleted with the total number of entries on the page and perform checks based on that

Reply:
> Does it cover the followings: Lets say 4 pages. i) delete few or all from first page ii) delete few or all from second, third page iii) delete few or all from last page iv) select few or all but no delete and just the navigation to back and forth of pages and check the scenarios.

The table still shows the correct data since we are performing pagination based on the total count of entries and the number of entries per page, But yes, I tried these scenarios and it is still functioning properly